lambda-notes/src/components/DeleteModal.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+
+const DeleteModal = props => {
+  const id = props.match.params.id;
+    const deleteIt = () => {
+      props.deleteNote(id);
+      props.history.push('/');
```

Comment:
same here with a race condition . Same warning . 
could be a hack but if you want you could try setTimeout to instantiate history.push( )?
toggling your notesSaved is causing a conflict